oteapi/models/datacacheconfig.py

```diff
@@ -10,13 +10,15 @@
 
 class DataCacheConfig(AttrDict):
     """DataCache Configuration.
-
     This class should not be used directly as a configuration object
```

Comment:
Revert the deletion, essentially.
```suggestion

    This class should not be used directly as a configuration object
```